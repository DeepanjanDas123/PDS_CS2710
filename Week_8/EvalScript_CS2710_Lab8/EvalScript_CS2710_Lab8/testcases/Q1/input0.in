4
7
push_back 10
push_front 20
push_back 30
push_front 40
pop_front
pop_back
isEmpty