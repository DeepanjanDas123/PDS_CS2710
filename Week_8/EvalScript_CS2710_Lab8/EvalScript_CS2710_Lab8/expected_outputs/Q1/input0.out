40
30
False